src/sentry/utils/pytest/relay.py

```diff
@@ -94,7 +96,7 @@ def relay_server_setup(live_server, tmpdir_factory):
     container_name = _relay_server_container_name()
     _remove_container_if_exists(docker_client, container_name)
     options = {
-        "image": "us.gcr.io/sentryio/relay:nightly",
+        "image": f"us.gcr.io/sentryio/relay:{RELAY_DOCKER_TAG}",
```

Comment:
Does relay push a new tag for PRs?

Reply:
I don't know; why?

Reply:
What would RELAY_DOCKER_TAG be then?

Reply:
In the Relay PR, I set the variable to match the Github sha:
<img width="329" alt="image" src="https://user-images.githubusercontent.com/44410/126514374-ca0810f4-b4b0-4aca-be8e-ca513b579bc4.png">

For normal execution, it will default to `nightly`.

Reply:
Yeah, I'm wondering if on the relay CI side of things if we push to GCR the `github.sha` tag